src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/GoogleCalendarService.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using System.Transactions;
+using DevAdventCalendarCompetition.Services.Interfaces;
+using DevAdventCalendarCompetition.Services.Models;
+
+namespace DevAdventCalendarCompetition.Services
+{
+    public class GoogleCalendarService : IGoogleCalendarService
+    {
+        private readonly HttpClient _httpClient;
+
+        public GoogleCalendarService(HttpClient httpClient)
+        {
+            this._httpClient = httpClient;
+        }
+
+        public async Task<CalendarList> GetAllCalendars()
+        {
+            return await this._httpClient.GetFromJsonAsync<CalendarList>("users/me/calendarList");
+        }
+
+        public async Task<string> CreateNewCalendarWithEvents(string calendarSummary, DateTime startDate, DateTime endDate)
+        {
+            var calendarResponse = await this.CreateCalendar(calendarSummary);
+
+            // For debug
+            calendarResponse.EnsureSuccessStatusCode();
+
+            var newCalendar = await calendarResponse.Content.ReadFromJsonAsync<CalendarDto>();
+
+            var eventsResponse = await this.CreateEvents(newCalendar.Id, startDate, endDate);
+            eventsResponse.EnsureSuccessStatusCode();
+
+            if (calendarResponse.IsSuccessStatusCode && eventsResponse.IsSuccessStatusCode)
+            {
+                return $"Pomyślnie utworzono kalendarz: {calendarSummary}";
```

Comment:
please return the success response, define the message in the resource file, and use it in the Controller